packages/calling/src/common/Utils.ts

```
@@ -1580,17 +1585,63 @@ export function modifySdpForIPv4(sdp: string): string {
  *
- * @param webex - Webex object to get service urls.
- * @param data - Data to be uploaded.
+ * @param metaData - Metadata to be uploaded.
+ * @param throwError - Whether to throw exception on failure (default: false).
+ * @returns Promise containing upload response if successful.
  */
-export async function uploadLogs(data = {}) {
+export async function uploadLogs(
+  metaData: LogsMetaData = {},
+  throwError = false
+): Promise<UploadLogsResponse | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)